packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/ResultTableView/InfiniteTable.js

```diff
@@ -87,6 +90,10 @@ export class InfiniteTable extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    if (!this.list) {
+      return
```

Comment:
this is not ok because there are other things happening in here that maybe shouldn't be skipped just because `this.list` is not set. you should add the check `this.list &&` in line 98 then